src/pages/pricing.tsx

```diff
@@ -266,7 +266,7 @@ const PricingPage: React.SFC<{}> = () => (
                 <ActionCard
                     title='Any Questions?'
                     text='We’re happy to answer them. Please get in touch.'
-                    anchors={[{href: '/contact', text: 'Contact'}]}
+                    anchors={[{href: '/contact', subject: "I have a question regarding Pricing", text: 'Contact'}]}
```

Comment:
@nisarhassan12 Please note that the canonical URL is `/contact/` and not `/contact`.

Linking to just `/contact` will cause users to go through an additional HTTP round-trip due to the redirection.

```suggestion
                    anchors={[{href: '/contact/', subject: "I have a question regarding Pricing", text: 'Contact'}]}
```